config/opendatahub.json

```diff
@@ -0,0 +1,24 @@
+{
+    "feed_data": {
+        "system_information": {
+            "email": "hello@AlpsGo.it",
+            "feed_contact_email": "mail@leonard.io",
+            "language": "de",
+            "license_url": "https://creativecommons.org/publicdomain/zero/1.0/legalcode",
+            "license_id": "CC0",
+            "attribution_organization_name": "AlpsGo Srl",
+            "attribution_url": "https://www.alpsgo.it",
+            "name": "Alps Go!",
+            "operator": "AlpsGo Srl",
+            "phone_number": "+39 0471 1390 502",
+            "privacy_url": "https://www.alpsgo.it/en/privacy",
+            "privacy_last_updated": "2025-06-07",
+            "purchase_url": "https://www.alpsgo.it",
+            "system_id": "alps-go",
```

Comment:
I suggest to name the config according to the system_id. For every car sharing service, one GBFS feed should be generated and a corresponding config should provide system_information etc.